### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DukeDavis12**
Update README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,13 +17,5 @@ The following repositories are created to support implementations for SDO protoc
 | [rendezvous-service](https://github.com/secure-device-onboard/rendezvous-service) | JAVA-based reference implementation for Rendezvous component defined in SDO protocol. |
 | [release](https://github.com/secure-device-onboard/release-fidoiot) | Hosts binary artifacts of different tagged releases of FDO protocol implementations. |
```
Comment (v-vasavi) [inline README.md]: is this SDO or FDO?

**Review by v-vasavi — COMMENTED** [review]
Do you think we need to have a copy of EPID here?

**Review by v-vasavi — COMMENTED** [review]
Do you think we need to have a copy of EPID here?

**Comment by DukeDavis12** [comment]
> Do you think we need to have a copy of EPID here?

It's not part of secure-device-onboard org. I think it's better not to add it here.

Let's say if we fork epid-verification-service in SDO org; Then we need to maintain 2 separate repos.

**Review by v-vasavi — APPROVED** [review]

**Review by v-vasavi — APPROVED** [review]

**Review by v-vasavi — APPROVED** [review]

**Review by KiranSukhavasi — APPROVED** [review]

**Review by Sai-Anudeep47 — APPROVED** [review]

**Review by bprashan — APPROVED** [review]

**Pull request merged**